src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSpec.scala

```diff
@@ -267,5 +294,87 @@ class LibraryServiceSpec extends FreeSpec with LibraryServiceSupport {
         }
       }
     }
+    "in its runtime schema definition" - {
+      "has valid JSON" in {
+        val fileContents = FileUtils.readAllTextFromResource("library/attribute-definitions.json")
+        val jsonVal:Try[JsValue] = Try(fileContents.parseJson)
+        assert(jsonVal.isSuccess, "Schema should be valid json")
+      }
+      "has valid JSON Schema" in {
+        val schemaStream = new URL("http://json-schema.org/draft-04/schema").openStream()
```

Comment:
I worry that referencing an external, unmanaged resource will introduce some flakiness to the test. I suppose if it becomes an issue, we can add a local copy of the schema to use.